### ISSUE TITLE
$parent or $parent.$parent does not work in firefox

### ISSUE THREAD

**Issue by devmondo** [aurelia/templating#134]
this code works in chrome fine but not in firefox i also tried ```trigger``` but same result
```
    <button type="button" class="btn btn-danger"  click.delegate="$parent.RemoveException($index)">
     remove
     </button>
```
the code above is inside repeat and no custom elements in use

**Comment by EisenbergEffect** [comment]
What happens exactly? Do you get an error? or does it just not trigger the method?

**Comment by devmondo** [comment]
@EisenbergEffect  no errors, just dont trigger it as you said :)

**Comment by devmondo** [comment]
ok, i can confirm now it is not working with IE too

**Comment by martingust** [comment]
@devmondo What version of `templating`is this? I just tested with v0.13.1 and the latest v0.13.11 and it worked in Firefox 39.0.

**Comment by devmondo** [comment]
@martingust  i did clean install and updated to latest, still same issue, also my friend on his machine facing same issue

**Comment by devmondo** [comment]
@martingust  i found the issue it is related to my other issue
https://github.com/aurelia/router/issues/162#issuecomment-122752504

once i removed it, ```$parent``` works!

**Comment by EisenbergEffect** [comment]
Fixed in 7764774cf6621d25a111759cbe1217dfae64812d

**Comment by devmondo** [comment]
:+1:
